components/x-article-save-button/readme.md

```diff
@@ -0,0 +1,41 @@
+# Article Save Button
+
+This module provides a template for myFT save article buttons, and is intended to replace and is intended to replace the legacy handlebars component in [n-myft-ui](https://github.com/Financial-Times/n-myft-ui/tree/master/components/save-for-later).
+
+## Installation
+
+```bash
+npm install --save @financial-times/x-article-save-button
+```
+
+## Props
+
+(Some of the properties don't influence the way button looks or acts, but can be used for e.g. client-side Javascript in the apps).
+	
+Feature                     | Type    | Required | Default value      | Description
+----------------------------|---------|----------|--------------------|----------------------
+`contentId`                 | String  | yes      | none               | UUID of the article
+`contentTitle`              | String  | yes      | none               | Title of the article
+`saved`                     | Boolean | no       | `false`            | Whether the article is saved or not.
+`csrfToken`                 | String  | no       | none               | CSRF token (will be included in a hidden form field).
```

Comment:
How can I get one of these?

Reply:
I took the view that this component should be agnostic about that. Where the CSRF token comes from (if one is needed at all) would vary depending on the application that consumes the button, and could change in the future.